src/main/scala/com/datastax/cdm/job/GuardrailCheck.scala

```diff
@@ -22,10 +22,6 @@ object GuardrailCheck extends BasePartitionJob {
 
   protected def execute(): Unit = {
     if (!parts.isEmpty()) {
-      originConnection.withSessionDo(originSession =>
```

Comment:
Reviewers assist: This was incorrectly added earlier to the tracking feature, its never tracked & hence removed it; only job types that are tracked are Migrate & DiffData.